Upper bounds shifts due to extra warnings re. internet connectivity.

diff --git a/tests/reg-packages.R b/tests/reg-packages.R
index c9962ce..a40b0fa 100644
--- a/tests/reg-packages.R
+++ b/tests/reg-packages.R
@@ -260,7 +260,7 @@ stopifnot(exprs = {
     (lenN <- length(print(iN <- grep("^[1-9][0-9]:", tlines)))) >= 2
     iN - iw == seq_len(lenN) # these (3) lines come immediately after 'Warning',
     ## and "related" to the some 'missing .. paren' above:
-    8 <= print(iw - i) & iw - i <= 20 # see ~14
+    8 <= print(iw - i) & iw - i <= 22 # see ~14
 }) ## failed in R <= 4.1.1


